packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -57,14 +57,6 @@ abstract class InspectorServiceBase extends DisposableController
         inspectorLibrary = EvalOnDartLibrary(
           inspectorLibraryUri,
           serviceManager.service!,
-          // TODO(jacobr): evaluate whether oneRequestAtATime is really required.
-          // The out of order request issues seen may have been isolated to Java
-          // where requests could truly be out of order due to multiple threads.
-          // It appears that enforcing in-order requests has significant negative
-          // consequences that out-weigh the benefits of being able to cancel
-          // requests from object groups that have been disposed before the requests
-          // were issued.
-          oneRequestAtATime: true,
           isolate: serviceManager.isolateManager.mainIsolate,
```

Comment:
we will want to make sure that the # of exceptions from the inspector does not go up after this release goes public. oneRequestAtATime. Removing this should help performance but may result in a lot more bugs along the lines of "inspector object id already disposed"